experimental/traffic-portal/.eslintrc.json

```diff
@@ -88,7 +88,7 @@
 				],
 				"id-match": "error",
 				"import/no-deprecated": "warn",
-				"jsdoc/newline-after-description": "error",
+				"jsdoc/newline-after-description": "off",
```

Comment:
Oh, this one too.

Reply:
sure, you won't see this

Reply:
So looking at the output, it seems they've gotten rid of this rule. [The release notes for the plugin version that dropped it](https://github.com/gajus/eslint-plugin-jsdoc/releases/tag/v42.0.0) mention a way to achieve equivalent configuration using [the `tag-lines` rule](https://github.com/gajus/eslint-plugin-jsdoc/blob/3a5dd7d9f67fc01ec848758443c04da29d45327b/docs/rules/tag-lines.md#readme). I'm afraid you're going to have to replace this line, actually. I didn't realize that this rule had been removed earlier - my bad.

Reply:
I updated the eslint